### DISCUSSION TITLE
Will frigate every support multi-cam traditional timeline scrubbing?

### DISCUSSION THREAD

**Discussion by Anaxagoras-bc** [blakeblackshear/frigate#9399] [General]
I get it goes against the ethos of "object detection" and getting rid of the noise, but frankly, having a full day timeline view is sometimes just necessary.

Example, i live on a busy street, a lot of people walk by every day, and a lot of cars go by.   So the first argument someone might make is, why am i recording the front street?  I used to only record my property and not the sidewalk, and the street, until my catalytic converter was stolen one night when i had to park out front, but had that area masked out to cut down on event noise.  I realize i would have never ID'ed them, but still, I would have liked to of at least had the opportunity.

So this week I had a package stolen because the fedex driver loves to leave packages at the base of my front steps, practically on the side walk.  So i had to click through what felt like close to 100 videos to find when a package was stolen after it was delivered.  With a traditional timeline scrub it really would have been a hell of a lot easier.  The experimental UI, is OK, but not great.

Scrubbing multiple camera's at once would be even better because i have multiple cameras in front of my house, looking at the other angles i was able to see his vehicle much better, but seeing both angles at the same time would have just made life easier.

**Comment by NickM-27** [comment]
the ui is being rewritten in 0.14 and there are a number of new features planned including a way to very quickly scrub footage of a camera

**Reply by burner-** [reply]
fyi, I recommend to look how unifi did that ui. They collect motion score and show it as "volume" graph on scrub bar. It really helps to scrub relevant locations.

**Reply by NickM-27** [reply]
The timeline for motion review has motion levels shown

**Reply by burner-** [reply]
perspective of future AI support and so on also is good to keep mind general augmentation support. I mean that api what allows add icons with hoverable descriptions to timeline. That open oppurtunity to external implementations so that example home automation motion sensors and external ai can add events to same timeline. That will push frigate to ahead of other VDR solutions.
